documentation/modules/ROOT/pages/configuration/topic-routing.adoc

```diff
@@ -56,79 +84,112 @@ transforms.Reroute.topic.regex=(.*)customers_shard(.*)
++
+In the example, the regular expression, `(.*)customers_shard(.*)` matches records for changes to tables whose names include the `customers_shard` string. This would re-route records for tables with the following names:
++
+`myserver.mydb.customers_shard1` +
+`myserver.mydb.customers_shard2` +
+`myserver.mydb.customers_shard3`
 
-== Key Fields
+`topic.replacement`:: Specifies a regular expression that represents the destination topic name. The transformation routes each matching record to the topic identified by this expression. In this example, records for the three sharded tables listed above would be routed to the `myserver.mydb.customers_all_shards` topic. 
 
-To address the concern of uniqueness across all the original tables discussed above,
-one more field for identifying the original (physical) table may be inserted into the key structure of the change events.
-By default, this field is named `\__dbz__physicalTableIdentifier` and has the original topic name as its value.
+ifdef::community[]
+== Ensure unique key
+endif::community[]
 
-If your tables already contain globally unique keys and you do not need to change the key structure, you can set the `key.enforce.uniqueness` property to `false`:
+ifdef::product[]
+// Type: procedure
+[id="ensuring-unique-keys-across-records-routed-to-the-same-topic"]
+== Ensuring unique keys across records routed to the same topic
+endif::product[]
 
-[source]
-----
-...
-transforms.Reroute.key.enforce.uniqueness=false
-...
-----
+A {prodname} change event key uses the table columns that make up the table's primary key. To route records for multiple physical tables to one topic, the event key must be unique across all of those tables. However, it is possible for each physical table to have a primary key that is unique within only that table. For example, a row in the `myserver.mydb.customers_shard1` table might have the same key value as a row in the `myserver.mydb.customers_shard2` table. 
+
+To ensure that each event key is unique across the tables whose change event records go to the same topic, the `ByLogicalTableRouter` transformation inserts a field into change event keys. By default, the name of the inserted field is `+__dbz__physicalTableIdentifier+`. The value of the inserted field is the default destination topic name.
 
-If needed, another _field name_ can be chosen by means of the `key.field.name` property
-(obviously you'll want to choose a field name that doesn't clash with existing primary key fields).
-For example the following configuration will use the name `shard_id` for the key field:
+If you want to, you can configure the `ByLogicalTableRouter` transformation to insert a different field into the key. To do this, specify the `key.field.name` option and set it to a field name that does not clash with existing primary key field names. For example: 
```

Comment:
Was the example configuration for `transforms.Reroute.key.enforce.uniqueness` and its descriptive paragraph intended to be omitted?  This is the specific section from master:

```
If your tables already contain globally unique keys and you do not need to change the key structure, you can set the `key.enforce.uniqueness` property to `false`:

[source]
----
...
transforms.Reroute.key.enforce.uniqueness=false
...
----
```